spec/models/conference_spec.rb

```diff
@@ -7,6 +7,22 @@
   it { is_expected.to have_many(:attendees) }
   it { is_expected.to validate_presence_of(:name) }
   it { is_expected.to validate_presence_of(:round) }
+  it { is_expected.to validate_presence_of(:starts_on) }
+  it { is_expected.to validate_presence_of(:ends_on) }
+
+  describe 'it checks the chronological order of dates' do
+    subject { FactoryGirl.build(:conference) }
```

Comment:
:thinking: It looks is doesn't after all. 
Without `subject.valid?` `subject.errors[:ends_on]` is always empty. 
I can either add `subject.valid?`, or do 
`expect(subject).not_to be_valid`. Does it matter?